configuration/outbound.md

```diff
@@ -0,0 +1,201 @@
+# Outbound push [beta]
+
+*Outbound is only available in Medic 3.5.0 and above*
+
+Outbound push allows configurers to have single reports generated by CHWs or others trigger outbound REST requests using the data in that report. For example, upon receiving a referral report you could send that referral to an external facility system that will manage and process that event.
+
+This feature is experimental, and may be replaced by some third party technology over time as we discover the feature-set we're looking for.
+
+## Configuration
+
+For outbound pushes to occur, you must [enable the `mark_for_outbound` transition in config](./transitions.md#Configurations):
+```json
+{
+  "transitions": {
+    "mark_for_outbound": true
+  }
+}
+```
+
+The rest of the configuration is against the `outbound` configuration property, which allows you to configure as many outbound pushes as you like, keyed off a unique name:
+
+```json
+{
+  "outbound": {
+    "first config": {
+      "relevant_to": "...",
+      "destination": {},
+      "mapping": {}
+    },
+    "second config": { }
+  }
+}
+```
+
+Each outbound push configuration contains the following properties: `relevant_to`, `destination` and `mapping`.
+
+### relevant_to
+
+An "expression" (some JS code that resolves to a truthy or falsy value) that determines whether this configuration is relevant to the document that is being tested. The document is passed to the expression is `doc`, and is fully hydrated before being passed.
+
+Example: you want to send a referral to an external facility tool when a CHW refers a patient:
+
+```json
+{
+  "relevant_to": "doc.type === 'data_record' && doc.form === 'referral'"
+}
+```
+
+### destination
+
+A complex property that declares how the data should be passed to the external service. It currently supports two authentication types, Basic Auth and a custom authentication mode for Muso SIH.
```

Comment:
`how the data should be passed to the external service`

Maybe something like "defines the details of the connection to the external service"